spectec/src/backend-latex/render.mli

```diff
@@ -5,17 +5,24 @@ open El.Ast
 
 type env
 
-val env : Config.t -> El.Ast.script -> env
+val env : Config.config -> El.Ast.script -> env
+val new_env : Config.config -> env
```

Comment:
Is this needed?

Reply:
Just some clutter that I forgot to remove, thanks!